index.html

```diff
@@ -278,11 +292,11 @@ <h5>Skills</h5>
                     </ul>
                 </div>
             </div>
-
+        </div>
         </div>
     </section>
     <section>
```

Comment:
- [ ] **Required:** Great job so far! I observed that the following changes still need to be fixed to get you to the next project. The below numbering is marked on your design image attached:

1.  The text in this section is aligned to the centre at the moment. Let's change it to left align.
2. All items here are all aligned to the left as well. as shown in the image.
3. There is a horizontal line before the social icons.
4. Place the `textarea` in front of the SVG to be similar to the Figma template.
5. Add more padding between the form and the footer social icons.

Your Design | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/150633789-33667493-06f0-4c9f-89d8-bdbcba8c3286.png) | ![image](https://user-images.githubusercontent.com/27709832/150633654-44801636-c388-41ba-a37f-5c168938442f.png)

I hope this helps 😄

Reply:
Hello @brytebee 
Thanks for your review 😄